### DISCUSSION TITLE
Create Super User

### DISCUSSION THREAD

**Discussion by ScriptKrewe** [nickjj/docker-django-example#47] [General]
Maybe I am missing something but how do you create the super user for admin access? I could not find any where that is being set.

**Comment by ScriptKrewe** [comment]
I figured it out. Thanks.

**Reply by nickjj** [reply]
Nice work. Did you end up running `./run manage createsuperuser`? That would be the shortcut to run any `manage` command.

**Reply by ScriptKrewe** [reply]
Yeah, I just read through the run file. This is a very sweet setup you have created. Thank you!
